help/status-and-availability.md

```diff
@@ -45,6 +55,44 @@ You can set a status emoji, status message, or both.
 
 {end_tabs}
 
+### Remove a status
+
+You can remove a status emoji, status message, or both.
+
+{start_tabs}
+
+{tab|via-user-card}
+
+{!self-user-card.md!}
+
+1. Click **Set a status**.
+
+1. Click the <kbd>X</kbd> beside the status
```

Comment:
This isn't a keyboard shortcut. There should be other examples in the help documentation where we use the icon in an instruction.